test/bootstrap.jl

```diff
@@ -34,7 +35,7 @@ include("modelcache.jl")
         # restore the original state
         refit!(fm, vec(float.(ds.yield)))
     end
-
+@static if VERSION < v"1.7.0-DEV.700"
```

Comment:
can we remove these guards and/or open an issue reminding us to do so when the type inference issue is solved in Julia?

Reply:
Please